repo2docker/buildpacks/conda/requirements.py-3.5.txt

```diff
@@ -0,0 +1,17 @@
+# pip-installed packages for Python 3.5
+# these should only be packages
+# whose versions are too new to have builds available for Python 3.5
+alembic==1.0.3
```

Comment:
This is exactly the `pip:...` lines from the previous environment.py-3.5.frozen.yml